app/assets/stylesheets/components/_quick-escape-banner.scss

```diff
@@ -1,23 +1,16 @@
 .app-c-quick-escape-banner {
-  @include govuk-font($size: 19, $weight: bold);
-
-  bottom: 0;
-  float: right;
-  padding: govuk-spacing(4) govuk-spacing(3);
-  position: fixed;
-  position: sticky; // scss-lint:disable DuplicateProperty
-  right: 0;
-
-  &:link,
-  &:visited,
-  &:hover,
-  &:active {
-    background-color: govuk-colour('black');
-    color: govuk-colour('white');
-  }
+  width: 100%;
+  position: -webkit-sticky; /* Safari */ // scss-lint:disable DuplicateProperty
+  position: sticky;
+  top: 0;
+  border-bottom: 1px solid govuk-colour("mid-grey");
+  background-color: govuk-colour("white");
+  z-index: 10000;
+  padding: govuk-spacing(3) 0 govuk-spacing(3);
+}
 
-  &:focus {
-    background-color: $govuk-focus-colour;
-    color: $govuk-focus-text-colour;
+.app-c-quick-escape-banner__link {
+  .gem-c-button {
+    @include govuk-font($size: 16, $weight: bold);
```

Comment:
A little bit, I did try to use a modifier but the component classes always overrode it. The other solution is that we remove the bold font but it may not have the correct level of emphasis that is needed for this.